oar/cli/cmd_change_advisory_status.py

```diff
@@ -1,11 +1,12 @@
-import click
 import logging
-from oar.core.worksheet import WorksheetManager
+
+import click
```

Comment:
Is it moved because the logging is from Python Standard Library? What tool do you use for the formatting the imports and the code?

Reply:
It seems just ConfigStore be removed

Reply:
@tomasdavidorg yes, exactly. I used built in IDE imports optimization.

Reply:
@ming1013 in this file, yes, only `ConfigStore` was removed.